packages/types/src/index.ts

```diff
@@ -472,9 +487,10 @@ export type Stop = {
   color?: string;
   dist?: number;
   geometries?: { geoJson?: GeoJSON.Polygon };
+  gtfsId: string;
   id: string;
-  lat: number;
-  lon: number;
+  lat?: number;
+  lon?: number;
```

Comment:
Why are these marked optional?

Reply:
I added them to make something work I think, but we should probably remove them and just expect these fields to always be there.